docs/variables.html

```diff
@@ -22,22 +22,46 @@
 <h1 class="title">Module <code>spin_sdk.variables</code></h1>
 </header>
 <section id="section-intro">
-<p>Module for getting Spin Variable values</p>
+<p>Module for interacting with Spin Variable</p>
```

Comment:
this everywhere _grin_

```suggestion
<p>Module for interacting with Spin Variables</p>
```